### ISSUE TITLE
Check individual copyright notes in doc strings

### ISSUE THREAD

**Issue by ahojnnes** [scikit-image/scikit-image#544]
Check individual copyright notes in doc strings and,  in case they are not needed, remove.

**Comment by stefanv** [comment]
This task is incomplete.

**Comment by alexdesiqueira** [comment]
@stefanv Debian's latest package [python3-skimage (0.18.3-3)](http://deb.debian.org/debian/pool/main/s/skimage/skimage_0.18.3-3.debian.tar.xz) contains this `copyright` file:

```
Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
Upstream-Name: scikit-image
Upstream-Contact: Stéfan van der Walt <stefan@sun.ac.za>
Source: https://github.com/scikit-image/scikit-image

Files: *
Copyright: 2009-2018 the scikit-image team
License: BSD-3-Clause

Files: skimage/external/tifffile/tifffile.py
Copyright: 2008-2014 Christoph Gohlke
           2008-2014 The Regents of the University of California
License: BSD-3-Clause

Files: doc/source/themes/scikit-image/layout.html
Copyright: 2007-2010 the Sphinx team
License: BSD-3-Clause

Files: skimage/filters/_ctmf.pyx skimage/filters/edges.py
       skimage/filters/_rank_order.py
       skimage/morphology/heap_general.pxi
       skimage/morphology/heap_watershed.pxi
       skimage/morphology/_skeletonize.py
       skimage/morphology/tests/test_watershed.py
       skimage/morphology/_watershed.pyx
       skimage/morphology/watershed.py
       skimage/feature/_canny.py
Copyright: 2003-2009 Massachusetts Institute of Technology
           2009-2011 Broad Institute
	   2003-2005 Peter J. Verveer
License: BSD-3-Clause

Files: skimage/filters/thresholding.py skimage/graph/_mcp.pyx skimage/graph/heap.pyx
Copyright: 2009-2015 Board of Regents of the University of
           Wisconsin-Madison, Broad Institute of MIT and Harvard, and Max Planck
           Institute of Molecular Cell Biology and Genetics
	   2009 Zachary Pincus
	   2009 Almar Klein
License: BSD-2-Clause

Files: debian/*
Copyright: 2010-2016 Yaroslav Halchenko <debian@onerussian.com>
           2018 Yangfl <mmyangfl@gmail.com>
	   2019-2020 Ole Streicher <olebole@debian.org>
License: BSD-3-Clause

License: BSD-2-Clause
 Redistribution and use in source and binary forms, with or without
 modification, are permitted provided that the following conditions
 are met:
 1. Redistributions of source code must retain the above copyright
    notice, this list of conditions and the following disclaimer.
 2. Redistributions in binary form must reproduce the above copyright
    notice, this list of conditions and the following disclaimer in the
    documentation and/or other materials provided with the distribution.
 .
 THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
 ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
 LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
 A PARTICULAR PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE HOLDERS OR
 CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
 EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
 PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
 PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
 LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
 NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

License: BSD-3-Clause
 Redistribution and use in source and binary forms, with or without
 modification, are permitted provided that the following conditions
 are met:
 1. Redistributions of source code must retain the above copyright
    notice, this list of conditions and the following disclaimer.
 2. Redistributions in binary form must reproduce the above copyright
    notice, this list of conditions and the following disclaimer in the
    documentation and/or other materials provided with the distribution.
 3. Neither the name of the University nor the names of its contributors
    may be used to endorse or promote products derived from this software
    without specific prior written permission.
 .
 THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
 ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
 LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
 A PARTICULAR PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE HOLDERS OR
 CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
 EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
 PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
 PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
 LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
 NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

Let me know what portions of it we would need and where, and I'll do it.

**Comment by stefanv** [comment]
Yup, that's what we need.

**Comment by alexdesiqueira** [comment]
Fixed in #6419. Thanks everyone!
